### PR TITLE
#14 added naming conventions as discussed in FHIR TC Calls

### PR DESCRIPTION
**Pull request by OliverKrauss**
From the summary in #14 the naming conventions were added to CONTRIBUTING.md, including some examples for understanding. In the next call where we have time -> If vote succeeds merge into master.

### CODE REVIEW

**Comment by Reitstaetter** [comment]
Sieht gut aus...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Contributing to hl7-at-fhir-profiles
 
-The following is a set of guidelines for contributing to the hl7-at-fhir-profiles project and its packages, 
-which are hosted in the HL7-Austria Organization on GitHub. These are mostly guidelines, not rules. Use your best judgment, 
+The following is a set of guidelines for contributing to the hl7-at-fhir-profiles project and its packages,
+which are hosted in the HL7-Austria Organization on GitHub. These are mostly guidelines, not rules. Use your best judgment,
```
[the remark this answers is not in the document]
Comment (OliverKrauss) [reply CONTRIBUTING.md]: Fixed this typo, and others.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -71,11 +71,75 @@ A commit message must start with the correspoinding ticket number in GitHub (#TI
...
+-  be U.S. English (spelled correctly!)
+-  be expressed as a noun, with a preceding adjective where necessary to clarify the semantics and to make unique
+-  not make use of trade-marked terms
+
+#### Profile Naming conventions
+
+A profile follows a prefix pattern, meaning that a name from left to right goes from specific to generic. It uses UpperCamelCase.
+
+**ProfileName** = [*Realm*-] *Use* , *ParentProfile*
+**Realm** = Is this profile supposed to be used in a realm? Then use the **countryCode**
```
[the remark this answers is not in the document]
Comment (OliverKrauss) [reply CONTRIBUTING.md]: Sooooo. Turns out ISO 3166 actually calls this "countryCode" so I did NOT change the name. I did however add a footnote that references the appropriate ISO and specifies Alpha-2 all lower case

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (oliver-krauss) [reply CONTRIBUTING.md]: Looks good in the new version Thanks!

[the remark this answers is not in the document]
Comment (OliverKrauss) [reply CONTRIBUTING.md]: Happy to help.

[the remark this answers is not in the document]
Comment (oliver-krauss) [reply CONTRIBUTING.md]: Please do NOT resolve the conversation. Only the reviewer should resolve so they can actually check if the fix was applied as asked for.

[the remark this answers is not in the document]
Comment (OliverKrauss) [reply CONTRIBUTING.md]: I will remember that for next time thanks

File: CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (oliver-krauss) [reply CONTRIBUTING.md]: Ok for me!

**Pull request merged**